src/ansys/mapdl/core/launcher.py

```diff
@@ -1438,6 +1444,40 @@ def launch_mapdl(
         "ANSYSLMD_LICENSE_FILE":"1055@MYSERVER"}
     >>> mapdl = launch_mapdl(replace_env_vars=my_env_vars)
     """
+    # By default
+    ON_SLURM = os.environ.get("PYMAPDL_ON_SLURM", None)
+    if ON_SLURM is None:
+        ON_SLURM = True
+    else:
+        # Unless the env var is false, it will be true.
+        ON_SLURM = not (ON_SLURM.lower() == "false")
+
+    # Let's require the following env vars to exists to go into slurm mode.
+    ON_SLURM = (
+        ON_SLURM
+        and bool(os.environ.get("SLURM_JOB_NAME", ""))
+        and bool(os.environ.get("SLURM_JOB_ID", ""))
+    )
+
+    if detect_slurm_config and ON_SLURM:
+        LOG.info("On Slurm mode.")
+
+        # extracting parameters
+        exec_file, jobname, nproc, ram, additional_switches = _parse_slurm_options(
+            exec_file,
+            jobname,
+            nproc,
+            ram,
+            additional_switches,
+            **kwargs,
+        )
+        # To avoid timeouts
+        license_server_check = False
+        start_timeout = 2 * start_timeout
+        ON_SLURM = True  # Using this as main variable
+    else:
+        ON_SLURM = False
+
     if remove_temp_files is not None:
         warnings.warn(
             "The option ``remove_temp_files`` is being deprecated and it will be removed by PyMAPDL version 0.66.0.\n"
```

Comment:
```suggestion
            "The ``remove_temp_files`` option is being deprecated. It is to be removed in PyMAPDL version 0.66.0.\n"
```
I'm not sure if you were trying to avoid being specific by saying "by PyMAPDL version 0.66.0" because it could be before then? Perhaps "It is to be removed sometime before PyMAPDL version 0.66.0."